hudi-common/src/test/java/org/apache/hudi/common/model/TestDefaultHoodieRecordPayload.java

```diff
@@ -91,6 +91,34 @@ public void testActiveRecords(String key) throws IOException {
     assertEquals(payload2.combineAndGetUpdateValue(record1, schema, props).get(), record2);
   }
 
+  @ParameterizedTest
+  @MethodSource("org.apache.hudi.common.testutils.PreCombineTestUtils#configurePreCombine")
+  public void testActiveRecordsForOverwriteWithGreaterRecordPayload(String key) throws IOException {
```

Comment:
Rename the payload class as `FirstValueAvroPayload` and add a new test class `TestOverwriteWithLatestAvroPayload`.

Reply:
Addressd it @danny0405